docs/catalyst_101.md

```diff
@@ -136,6 +136,153 @@ Once the existing members have voted to offer you membership, it’s up to you t
 
 ## Project Management
 
+### Overall goals:
+
+- It should be easy to decide what to work on next.
+- We should feel some sense of progress in our day-to-day.
+- We should be able to identify dependencies and bottlenecks.
+- Each team should be able to communicate its goals on a two-week timeline to the other teams.
+- We should be able to stick to the process, because the overhead feels worth the effort.
+
```

Comment:
We should be able to break down our quarterly goals into smaller, logical steps and work towards them consistently over the quarter.

Reply:
This sounds like two things:
* we would like something between quarterly and sprint planning
* in the sprint we should be connecting the work we do to progress on our high level goals

I put some stuff in to that effect - let me know if I'm misinterpreting your words!

Reply:
Exactly it!